react/components/EXPERIMENTAL_Charts/LineChart/index.tsx

```diff
@@ -67,10 +69,13 @@ LineChart.propTypes = {
   xAxisKey: PropTypes.string.isRequired,
   
   /** The formatter function of value in tooltip. If you return an array, the first entry will be the formatted "value", and the second entry will be the formatted "key" */
-  formatter: PropTypes.func,
+  tooltipFormatter: PropTypes.func,
   
   /** The schema prop changes some styles of the chart. This prop should be given as an object.*/
-  schema: PropTypes.object,
+  config: PropTypes.object,
```

Comment:
Can this object really have any shape? 🤔

Reply:
That would be a good one indeed.